woocommerce/woocommerce
# Monorepo utils: Fix GitHub flag

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Instead of depending on a `-g` option for running different behaviour on Github Actions, this PR changes the logic to look for the `CI` environment variable which is [always set to true on Github Actions](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables).



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run the verify day command as is. See the output.
```
pnpm run utils code-freeze verify-day
```
2. Run it again, now with the CI environment variable set. Compare the output and see that an output is logged, `::set-output name=freeze::false`
```
CI=true pnpm run utils code-freeze verify-day
```
